<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimeLog</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>361</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlMainLayout">
     <item>
      <layout class="QGridLayout" name="gLTimeCount">
       <item row="2" column="0">
        <widget class="QPushButton" name="pbWorkStartStop">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pbBreakStartStop">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lHeaderWorkTime">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Arbeitszeit</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lHeaderBreak">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lWorkTime">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lBreak">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vlProject">
       <item>
        <widget class="QComboBox" name="cbProjSel">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbProjComment">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlProject1">
         <item>
          <widget class="QLabel" name="lProjTime">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbProjStartStop">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>125</y>
      <width>155</width>
      <height>203</height>
     </rect>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Da&amp;tei</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/resource/menu.png</normaloff>:/resource/menu.png</iconset>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionDayView"/>
    <addaction name="actionReport"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <action name="actionOptions">
   <property name="text">
    <string>&amp;Optionen...</string>
   </property>
  </action>
  <action name="actionasdfsadf">
   <property name="text">
    <string>Tagesansicht</string>
   </property>
  </action>
  <action name="actionsadfsadf">
   <property name="text">
    <string>sadfsadf</string>
   </property>
  </action>
  <action name="actionEinstellungen">
   <property name="text">
    <string>Optionen</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>&amp;Übersicht...</string>
   </property>
  </action>
  <action name="actionDayView">
   <property name="text">
    <string>&amp;Tagesansicht...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Über TimeLog...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
